src/libraries/System.Net.Security/tests/FunctionalTests/NegotiateStreamStreamToStreamTest.cs

```diff
@@ -192,6 +193,11 @@ public async Task NegotiateStream_StreamToStream_Authentication_EmptyCredentials
         {
             string targetName = "testTargetName";
 
+            if (PlatformDetection.IsWindowsServer2025)
```

Comment:
This is a misnomer - it's checking any windows SKU newer than build 26100 -- so you've effectively turned this off these tests on all new windows machines.    I noticed as this was failing in release branches where we haven't added testing for Server 2025.